### ISSUE TITLE
PaperSpigot 1.16.5:  Console Filter doesn't work properly.

### ISSUE THREAD

**Issue by DotDotas** [kangarko/ChatControl#1280]
I was trying to remove this annoying message in console but it doesn't work.
![image](https://user-images.githubusercontent.com/87330727/131034729-0ba78a40-1a06-4310-b2eb-37de20355785.png)
I also tried to remove other messages in console but it doesn't work too.
![image](https://user-images.githubusercontent.com/87330727/131034863-7e7d5e57-77f3-4e6a-b1b7-e171609b6f82.png)
Minecraft Version: 1.16.5
MySQL: false
BungeeCord: false

**Comment by TheIntolerant** [comment]
This is actually normal behavior as WorldEdit and EssentialsX tend to load before chat control, therefore chat control doesn't have a chance to hide such messages as that plugin has loaded before chat control has loaded.

You can confirm the above messages are being hidden by chat control by running a command such as
``/say [Essentials] You are running an unsupported server version!`` if you cannot see it in the console but see it in-game this means the console filter is actually working as intended.

Attempting to filter messages sent to the console on startup using chat control will never work IF the message is by a plugin that has loaded before chat control has loaded, such as EssentialsX or WorldEdit. 

Basically you will only be able to filter messages sent to the console after chat control has loaded up.

**Comment by kangarko** [comment]
You can also actually grab our Tester plugin from mineacademy.org/plugins where you disable force op commands and other insecure stuff for production servers and use filtering feature from there since it is loaded much much earlier in the startup sequence.
